### PR TITLE
Feature/security page

### PR DESCRIPTION
**Pull request by invisnet**
First draft. PR created to provide a place for tracking feedback.

### CODE REVIEW

**Comment by Mte90** [comment]
I think that there are some issue with this pr because include a lot of commits that doesn't involve with pr

**Comment by invisnet** [comment]
It's only because the branch is behind develop - the PR to develop is clean.

File: src/wp-admin/security.php [hunk 1]
```diff
@@ -26,6 +51,106 @@
...
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="meta-box-sortables ui-sortable">
+          <div class="postbox">
+            <h2><span><?php _e( 'API' ); ?></span></h2>
+            <div class="inside">
+              <p>Notes: I left $capability for consistency; we should probably force it to 'manage_options' or just drop it completely - discuss.</p>
+              <!-- HTML generated using hilite.me -->
```
Comment (nylen) [inline src/wp-admin/security.php]: Which color scheme was used on hilite.me? Were there any modifications made to the resulting inline CSS?

File: src/wp-admin/security.php [hunk 2]
```diff
@@ -26,6 +51,106 @@
...
+          </div>
+        </div>
+        <div class="meta-box-sortables ui-sortable">
+          <div class="postbox">
+            <h2><span><?php _e( 'Usage' ); ?></span></h2>
+            <div class="inside">
+              <p>Lipsum</p>
+              <!-- HTML generated using hilite.me -->
+              <div style="background: #f8f8f8; overflow:auto;width:auto;border:solid gray;border-width:.1em .1em .1em .8em;padding:.2em .6em;">
+<pre style="margin: 0; line-height: 125%"><span style="color: #0000FF">add_security_page</span>(
```
Comment (nylen) [inline src/wp-admin/security.php]: Plugins that need to work in both CP and WP should do a `function_exists` check and register their setting somewhere else if running under WP (or older versions of CP). We should provide an example for this too.

File: src/wp-admin/security.php (hunk 1 above)
Comment (invisnet) [reply src/wp-admin/security.php]: default, no.

File: src/wp-admin/security.php (hunk 2 above)
Comment (invisnet) [reply src/wp-admin/security.php]: That and a load of other things, yes; I'm working down the page - why, what, how.

File: src/wp-admin/security.php (hunk 1 above)
Comment (nylen) [reply src/wp-admin/security.php]: It looks like the opening `<?php` tag needed to get hilite.me to colorize PHP code was removed. Let's document that, the name of the default color scheme, and any other steps needed to reproduce this example using hilite.me.

**Comment by nylen** [comment]
@invisnet: I've merged the latest changes from `develop` into the `feature/security-page` branch on this repository. Please rebase or resubmit your commits against `feature/security-page` so that we have a clean history here in this PR.

**Comment by invisnet** [comment]
Killing this, replaced by #484.

**Pull request closed without merging**